src/Sarif.Viewer.VisualStudio.Core/ErrorList/ErrorListService.cs

```diff
@@ -698,6 +703,61 @@ private int WriteRunToErrorList(Run run, string logFilePath, SarifLog sarifLog,
                 this.ShowFilteredSuppressionStateColumn();
             }
 
+            if (skipRemapping == false)
+            {
+                IEnumerable<string> relativeFilePaths = dataCache.SarifErrors.Select(x => x.FileName);
+                IEnumerable<string> uriBaseIds = dataCache.SarifErrors.Select(x => x.SarifResult.Locations?.FirstOrDefault()?.PhysicalLocation?.ArtifactLocation?.UriBaseId);
+
+                // now we need to map from relative file path to absolute.
+                string workingDirectory = dataCache.SarifErrors.FirstOrDefault().WorkingDirectory;
+
+                // find the mapped path with codeanalysisresultmanager
+                List<string> resolvedFilePaths = CodeManagerInstance.TryResolveFilePaths(dataCache, workingDirectory, logFilePath, uriBaseIds.ToList(), relativeFilePaths.ToList());
```

Comment:
>resolvedFilePaths [](http://example.com/codeflow?start=29&length=17)

What if this is empty?

Reply:
there is 1 entry in the resolvedFilePaths for every entry in the relativePaths as the foreach loop in TryResolveFilePaths adds something each iteration